app/src/main/java/ar/com/wolox/android/training/ui/login/LoginFragment.java

```diff
@@ -26,11 +38,27 @@ public int layout() {
 
     @Override
     public void setListeners() {
-        getView().findViewById(R.id.vTermsConditions)
-                .setOnClickListener(it -> getPresenter().onTermsAndConditionsClicked());
-        getView().findViewById(R.id.vLogInButton)
-                .setOnClickListener(it -> getPresenter().onLoginButtonClicked());
-        getView().findViewById(R.id.vSignUpButton)
-                .setOnClickListener(it -> getPresenter().onSignupClicked());
+        termsConditions.setOnClickListener(it -> getPresenter().onTermsAndConditionsClicked());
+        logInButton.setOnClickListener(it -> getPresenter().onLoginButtonClicked());
+        signUpButton.setOnClickListener(it -> getPresenter().onSignupClicked());
+    }
+
+    @Override
+    public void getSignUpView() {
```

Comment:
for the methods that show different screens instead of using `get[x]View` you can name them like `show[x]Screen` or `goTo[x]` screen